### PR TITLE
pokeCard color typeTags and basic styling

### LINKED ISSUE #15: Make a container PokeList to display filters and search result(Cards).
**Is your feature request related to a problem? Please describe.**
Cards require container for cards layout, filters component, infinite scroll.

**Describe the solution you'd like**
Make component, use flexbox for card layout, and scroll.

### LINKED ISSUE #17: Define a function to return card color.
**Is your feature request related to a problem? Please describe.**
Each pokemon card should have background color based on pokemon type.

**Describe the solution you'd like**
Define function that takes Pokemon type as arguments, and write conditional statements within to return color.

### PR DESCRIPTION
**Pull request by Gjhnsn**
## Changes
1. Created getTypeColors file
2. Implemented color coding for pokemon type tags
3. Basic styling to test functionality

## Purpose
_Describe the problem or feature in addition to a link to the issues._
Pokemon cards now have a color coded tag for their types

Closes #17 
Closes #15

### CODE REVIEW

File: src/components/Filter.js [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+    },
+    {
+      value: "steel",
+      label: "Steel",
+    },
+    {
+      value: "flying",
+      label: "Flying",
+    },
+  ];
```
Comment (BlackCubes) [inline src/components/Filter.js:135]: Maybe put this into its own file like allTypes.js in the utils folder?? Just a suggestion, but if it works then it works! 😃

**Review by BlackCubes — APPROVED** [review]
Looks pretty good! Good job on the functionalities!

File: src/components/Filter.js (hunk 1 above)
Comment (dannysanchez559) [reply src/components/Filter.js:135]: sounds good! i agree

Comment (Gjhnsn) [reply src/components/Filter.js:135]: Thanks! Will do with the refactor!

**Pull request merged**